rest/src/main/kotlin/service/InteractionService.kt

```diff
@@ -189,4 +192,51 @@ class InteractionService(requestHandler: RequestHandler) : RestService(requestHa
             keys[Route.GuildId] = guildId
             keys[Route.CommandId] = commandId
         }
+    suspend fun getGuildCommand(applicationId: Snowflake, guildId: Snowflake, commandId: Snowflake) = call(Route.GuildApplicationCommandGet){
```

Comment:
Added into the base branch already

Reply:
I think GitHub messed up here because I didn't add that method

Reply:
Looks like a GitHub highlighting issue I didn't add this

Reply:
I screwed up during rebase